src/parser/jobs/rpr/modules/HarvestMoon.tsx

```diff
@@ -0,0 +1,79 @@
+import {Trans} from '@lingui/react'
+import {DataLink} from 'components/ui/DbLink'
+import {Event} from 'event'
+import {Analyser} from 'parser/core/Analyser'
+import {filter} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import Checklist, {Requirement, TARGET, TieredRule} from 'parser/core/modules/Checklist'
+import {Data} from 'parser/core/modules/Data'
+import {Invulnerability} from 'parser/core/modules/Invulnerability'
+import {UnableToAct} from 'parser/core/modules/UnableToAct'
+import React from 'react'
+
+const SOULSOW_CAST_TIME = 5000 //milliseconds
+const SOULSOW_CAST_GRACE = 1000
+const USAGE_PERCENTAGE_SEVERITY = {
+	0: TARGET.FAIL,
+	100: TARGET.SUCCESS,
+}
+
+export class HarvestMoon extends Analyser {
+	static override handle = 'HarvestMoon'
```

Comment:
Alternatively lowercase the whole thing.
```suggestion
	static override handle = 'harvestMoon'
```